libp2p/kademlia/kad_peerinfo.py

```diff
@@ -16,9 +16,8 @@ class KadPeerInfo(PeerInfo):
     def __init__(self, peer_id, peer_data=None):
         super(KadPeerInfo, self).__init__(peer_id, peer_data)
 
-        self.peer_id_obj = peer_id
-        self.peer_id = peer_id.get_raw_id()
-        self.xor_id = peer_id.get_xor_id()
+        self.peer_id_bytes = peer_id.to_bytes()
```

Comment:
Will it be simpler to just store `ID`?

Reply:
Only peer ID(in bytes) is used in kad so I think we can just use `peer_id_bytes`.